static/app/views/dashboards/widgets/lineChartWidget/lineChartWidget.stories.tsx

```diff
@@ -88,3 +110,28 @@ export default storyBook(LineChartWidget, story => {
 const MediumWidget = styled('div')`
   width: 420px;
 `;
+
+function filterTimeSeriesToSelection(
```

Comment:
this does filtering + transformation to the time series :)
```suggestion
function toTimeSeriesSelection(
```